core/trino-main/src/main/java/io/trino/execution/TaskManagerConfig.java

```diff
@@ -79,11 +79,11 @@ public class TaskManagerConfig
     private Duration interruptStuckSplitTasksDetectionInterval = new Duration(2, TimeUnit.MINUTES);
```

Comment:
comment: some of the writers will do CPU and some IO, hence we need more writers